oryx-app-mllib/src/test/java/com/cloudera/oryx/app/mllib/kmeans/KMeansUpdateIT.java

```diff
@@ -58,8 +56,8 @@ public void testKMeans() throws Exception {
     overlayConfig.put("oryx.batch.streaming.generation-interval-sec", GEN_INTERVAL_SEC);
     overlayConfig.put("oryx.batch.streaming.block-interval-sec", BLOCK_INTERVAL_SEC);
     overlayConfig.put("oryx.kmeans.hyperparams.k", CLUSTERS);
-    overlayConfig.put("oryx.input-schema.num-features", 2);
-    overlayConfig.put("oryx.input-schema.numeric-features", "[\"0\",\"1\"]");
+    overlayConfig.put("oryx.input-schema.num-features", 5);
+    overlayConfig.put("oryx.input-schema.numeric-features", "[\"0\",\"1\",\"2\",\"3\",\"4\"]");
```

Comment:
Don't bother changing this, but you can also set cateogrical-features = [] to imply that everything is numeric. Yes this change is just fine of course.